ui/pager.go

```diff
@@ -267,13 +312,13 @@ func (m pagerModel) update(msg tea.Msg) (pagerModel, tea.Cmd) {
 		}
 
 	case spinner.TickMsg:
-		if m.state == pagerStateStashing || m.spinner.Visible() {
+		if m.state == pagerStateStashing { // TODO: || m.spinner.Visible() {
```

Comment:
@meowgorithm here I'll need some help probably, not sure how this worked before...

Reply:
Sounds good. If this PR is ready I can swoop in in a bit and take a look.

Reply:
fixing some linting issues, but it should be good to go :)